### DISCUSSION TITLE
Vercel Audiences Data Export

### DISCUSSION THREAD

**Discussion by shanefulmer** [vercel/community#1528] [Idea]
Is there any way to export all of the data? I'd like to see how many people have viewed a specific page.

**Comment by amyegan** [comment]
Thanks for reaching out, Shane! This is already on our radar from https://github.com/vercel/community/discussions/1068, but I'll update the internal ticket with your feedback.

For now, there are ways to change your view of the [Audience](https://vercel.com/docs/concepts/analytics/audiences#) data in your project dashboard. The timeframe can be adjusted by picking another option from the dropdown in the upper right hand corner. And you can use the [panels](https://vercel.com/docs/concepts/analytics/audiences#Panels) section to get a breakdown of page and visitor data.

**Comment by rcolepeterson** [comment]
How would one export the analytics as pdf?

**Reply by rcolepeterson** [reply]
I need to share the analytics data collected. How do i do that? Can't find an export button.

**Comment by ahlvnil** [comment]
I am in the same situation as @rcolepeterson - I want to share the analytics data with project people who are not necessarily Vercel users. Is there any current functionality for this, or something planned @amyegan?

**Comment by fmonper1** [comment]
This would be pretty useful for my use case too

**Comment by tyhouch** [comment]
Any update on this?

**Comment by amyegan** [comment]
The team is working on it with plans to make it available later this year.

Since this is a duplicate discussion, I'm going to close it. Please follow along in discussion https://github.com/vercel/community/discussions/1068, or in our [changelog](https://vercel.com/changelog), for updates as they become available. 🙏
